data/hai/unfettered side missions.txt

```diff
@@ -65,3 +66,406 @@ mission "Unfettered returning home"
+			`	"These legendary weapons can interfere with nearly any engine system! Worry not, you'll be able to recuperate the cost of this ship with ease by assisting us. A load of medical supplies is needed on <destination>, and we lack the time and personnel to bring them ourselves on such short notice.`
+			`	"Of course, the ship is aged, and you should not expect the best price in a shipyard. Nevertheless, it is a fine vessel, capable of fighting, freighting, and mining. So, what do you say, <last>? This is the opportunity of a lifetime; they simply don't make ships like this anymore."`
+			label choice
+			choice
+				`	"I will help you."`
+					goto take
+				`	"If you lack crew, who was piloting the other ship?"`
+					to display
+						not "unfettered: pilot dialog"
+					goto pilot
+				`	"Why was the other ship hidden underwater?"`
+					to display
+						not "unfettered: ocean dialog"
+					goto ocean
+				`	"No, thanks. I'm not looking to add another ship to my fleet."`
+			`	The two Hai look disappointed. "A shame. We will probably not meet again," one says. "It is best you forget we have met here," says the other. They signal the ship from earlier to submerge into the ocean again and walk off along the beach.`
+				goto "train back"
+			label pilot
+			action
+				set "unfettered: pilot dialog"
+			`	"The less you know, the better for all of us, Captain."`
+				goto choice
+			label ocean
+			action
+				set "unfettered: ocean dialog"
+			`	"We found it there," the first one answers.`
+			`	"It is hard to find a drop of rain in a storm, but the drop of rain may see all as it falls," the other one adds.`
+				goto choice
+			label take
+			action
+				give ship "Anomalocaris (Venerable)" "Good Omen"
+					id "hai legacy"
+				fine 10000000
+			`	They give you access to the ship and proceed to explain the details. "It is of no real significance, but surely you understand we cannot go through standard means for this sale. We will instead present it as a fine. You'll be charged for carrying technology you had no idea was forbidden, as these Korath weapons are suspicious indeed. However, it will not go on your permanent record."`
+			`	You quickly realize that they are not asking you for your opinion at all. In fact, you receive notice of the fine before they have even finished speaking. "We will need to make some modifications to the ship before you can use it.`
+			`	"Don't forget to return to the spaceport to get the cargo loaded onto your ship. You should use this ship and this ship alone to make these urgent deliveries."`
```

Comment:
It says I should use "this ship alone" for the the delivery, but that's not actually a requirement.

Reply:
I think this whole last line "Don't...deliveries." can be omitted, considering after this, the conversation continues automatically talking about you returning to the spaceport and all.

Reply:
```suggestion
```
I think (hope) they mention above that you should come only with this ship

Reply:
Then it kinda becomes a completely hidden thing the uhai dont fire on you
How about smth like this
```suggestion
			`	"A word of wisdom: the relics of old warrant respect, but not escorts."`
```

Reply:
It's not hidden; we have that `on enter` dialog when you enter their space that says they strangely stop shooting at you, so I think anything else is unecessary.

Reply:
Let it be a surprise to the player they stop shooting, no need to egg them on at first. Then it will make sense after the fact when they start talking about your precious relic ship.

Reply:
Oh I forgot they stop shooting even if you bring a fleet
```suggestion
```